archive/frames/views.py

```diff
@@ -190,8 +232,9 @@ class VersionViewSet(viewsets.ReadOnlyModelViewSet):
 
 
 class S3ViewSet(viewsets.ViewSet):
+    schema = ScienceArchiveSchema(tags=['Frames'])
 
-    @detail_route()
+    @action(detail=True)
     def funpack(self, request, pk=None):
         '''
         Download the given Version (one part of a Frame), run funpack on it, and
```

Comment:
Maybe modify this comment since it shows up in the docs now to make it clear the server will download the frame and unpack it and then serve up the unpacked file to the client.